src/rhsm_debug/debug_commands.py

```diff
@@ -233,7 +233,7 @@ def _do_command(self):
                 shutil.rmtree(content_path, True)
 
     def _make_code(self):
-        return datetime.now().strftime("%Y%m%d-%f")
+        return datetime.now().strftime("%Y%m%d-%H%M%S-%f")
```

Comment:
hmm I'd keep this change separate for now, as (for sad as it may look like) there could be users relying on this format

Reply:
I agree